addons/api/mirage/scenarios/default.js

```diff
@@ -127,6 +124,13 @@ export default function (server) {
     server.createList('role', 3, { scope });
   });
 
+  // Aliases
+  const aliases = server.createList('alias', 3, { scope: globalScope });
+
+  aliases.forEach((alias) => {
```

Comment:
I think this is not need.

`destination_id` is a constant that always return the same value and we are going through each alias to override an attribute that we know the value.

I will use the same attribute overrides, [link to docs](https://miragejs.com/docs/main-concepts/factories/#attribute-overrides), we are already using with `scope.`

for example:
```jacascript
const destination_id = server.schema.targets.all().models[0].id;
server.createList('alias', 3, { scope: globalScope, destination_id });
```

Reply:
ah yes good call, updated